docs/prow/prow-k8s-testgrid.md

```diff
@@ -1,73 +1,63 @@
+TestGrid configuration is stored inside [kubernetes/test-infra](https://github.com/kubernetes/test-infra/tree/master/config/testgrids/kyma) repository in the `/config/testgrids/kyma/` directory.
 
-## Kyma configuration
+TestGrid is being automatically updated by Prow using the tool called [transfigure.sh](https://github.com/kubernetes/test-infra/tree/master/testgrid/cmd/transfigure).
 
-One part of the configuration is based on job definitions in the form of annotations. See the example of the [`kind` job](https://github.com/kyma-project/test-infra/blob/60493dd61d77da363b8758b7e4c94f25d4b36501/prow/jobs/test-infra/test-infra-kind.yaml#L80-L83) displayed on the *kyma-nightly* dashboard on TestGrid:
+## Adding new dashboard
 
+The dashboards' configuration is stored in the [testgrid-default.yaml](https://github.com/kyma-project/test-infra/tree/master/prow/testgrid-default.yaml) file.
+This file is being automatically generated from a [template testgrid-default.yaml](https://github.com/kyma-project/test-infra/blob/master/templates/templates/testgrid-default.yaml) file.
 ```yaml
-  annotations:
-    testgrid-dashboards: kyma-nightly
-    # testgrid-alert-email: email-here@sap.com
-    testgrid-num-failures-to-alert: '1'
+dashboards:
+  # kyma
+  - name: kyma_integration
+  - name: kyma_control-plane
+
+  # kyma-incubator
+  - name: kyma-incubator_compass
+
+dashboard_groups:
+  - name: kyma
+    dashboard_names:
+      - kyma_integration
+      - kyma_control-plane
+      - kyma-incubator_compass
 ```
+To add a new dashboard it's needed to do several steps:
 
-You can also specify an email address if you want to receive notifications after a predefined number of job failures.
+1. Add a new dashboard definition in the `dashboards` field according to the example file above inside a template file. **Dashboards need to have dashboard name as prefix**.
+2. Add the previously added dashboard to the corresponding `dashboard_name` inside the `dashboard_groups` field.
+3. Generate a new config file using [rendertemplates](https://github.com/kyma-project/test-infra/tree/master/development/tools/cmd/rendertemplates) tool and check if the config file generated correctly.
 
-## Kubernetes configuration
+## Adding ProwJob to the TestGrid
 
-The other part of the configuration is stored in the [Kubernetes repository](https://github.com/kubernetes/test-infra/tree/master/config/testgrids). The Kyma team [owns](https://github.com/kubernetes/test-infra/blob/master/config/testgrids/kyma/OWNERS) the `kyma` folder which holds the configuration related to Kyma Prow jobs.
+After adding a desired dashboard in the `testgrid-default.yaml` file you need to add ProwJobs to the dashboard. It can be done by defining new `annotations` field to a ProwJob definition.
```

Comment:
```suggestion
After adding a desired dashboard in the `testgrid-default.yaml` file, add ProwJobs to the dashboard. To do so, define a new **annotations** field in a ProwJob definition.
```

Reply:
I think it would be best to leave the annotations word in a raw code brackets as it refers to the exact field which is placed in a prowjob definition.